client/src/redux/shopCart/cart-selectors.js

```diff
@@ -0,0 +1,33 @@
+import { createSelector } from "reselect";
+
+const selectCart = state => state.cart;
+
+export const selectCartItems = createSelector(
+  [selectCart],
+  cart => cart.cartItems
+);
+
+export const selectCartHidden = createSelector(
```

Comment:
`selectCartHidden` doesn't need a selector, it is here for the purpose of consistency

Reply:
Isn't this unused now?